pinot-plugins/pinot-input-format/pinot-json/src/main/java/org/apache/pinot/plugin/inputformat/json/JSONMessageDecoder.java

```diff
@@ -67,9 +66,4 @@ public GenericRow decode(byte[] payload, GenericRow destination) {
       return null;
     }
   }
-
```

Comment:
If offset/length is really used, instead of copying array in the super implementation, we could pass wrap payload in ByteArrayInputStream with offset and length, e.g. 

```java
  public static JsonNode bytesToJsonNode(byte[] jsonBytes, int offset, int length)
      throws IOException {
    return DEFAULT_READER.readTree(new ByteArrayInputStream(jsonBytes, offset, length));
  }
```